app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::SheltersController < ApplicationController
+  def index 
+    @shelters = Shelter.reverse_alpha_order 
+  end
+end
```

Comment:
This controller is great. Just wanted to talk about the `Admin::` piece a little. Since this is a nested controller inside of an `admin/` folder, this is the syntax to inform Rails of that. This is to differentiate it from the other `SheltersController` that isn't nested.